src/web/services/confirmation-modal.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { ConfirmationModalType } from '../app/components/confirmation-modal/confirmation-modal-type';
+import { ConfirmationModalComponent } from '../app/components/confirmation-modal/confirmation-modal.component';
+
+/**
+ * Service to handle display of confirmation modals.
+ * For more a complicated use case, use {@link NgbModal} directly.
```

Comment:
What kind of "more complicated use case" do we have right now?

Reply:
Modals like `CopyQuestionsFromOtherSessionsModalComponent` and `TemplateQuestionModalComponent` would be hard to fit into the standardized modal. i.e. modals that has non text only contents that involves callbacks and have local states.

Reply:
I think you can name the method as `openSimpleModel`.

Reply:
I see, that's right. Our target here is for simple enough modals, e.g. those that require no action or "yes/no" type of action.

Reply:
I renamed `confirmation-modal.service` to `simple-modal.service`, and then have 2 different methods `openConfirmationModal` and `openInformationModal` for modals that requires confirmation or no action respectively.